src/stars.css

```diff
@@ -0,0 +1,32 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
```

Comment:
The task description specifically mentions not to use the `gap` property in the `.stars` class. Although it's not used here, ensure that any spacing adjustments comply with this requirement.